opendatahub-io/odh-manifests
# Update Spark Operator to support ocp 4.9

Instructions to test:

- Deploy latest ODH Operator
- Create a namespace
- Add https://github.com/opendatahub-io/odh-manifests/blob/master/kfdef/kfctl_openshift.yaml to that namespace
- Spark operator should be up and running
